tools/sgsqlfluff/tools.go

```diff
@@ -26,6 +31,32 @@ func Command(ctx context.Context, args ...string) *exec.Cmd {
 	return sg.Command(ctx, sg.FromBinDir(name), args...)
 }
 
+func defaultConfigPath() string {
+	return sg.FromToolsDir(name, ".sqlfluff")
+}
+
+func CommandInDirectory(ctx context.Context, directory string, args ...string) *exec.Cmd {
+	configPath := filepath.Join(directory, "dbt/.sqlfluff")
```

Comment:
Do we require this file to be in your local repo? Maybe send in the full path to make it less tied to DBT?